docs/EN_US/ContainerizedHPCC/ContainerizedMods/LocalDeployment.xml

```diff
@@ -85,7 +85,10 @@
 
     <para><emphasis role="bold">To start this simple system:</emphasis></para>
 
-    <para><programlisting>helm install mycluster hpcc/hpcc --set global.image.version=latest </programlisting></para>
+    <para><programlisting>helm install mycluster hpcc/hpcc --version=8.2.2 </programlisting></para>
+
+    <para><emphasis role="bold">This example uses 8.2.2, but you should use
```

Comment:
I think somewhere it should mention needing to run: helm repo update
to get newer version fetched/available.